server/base/src/main/java/org/apache/accumulo/server/mem/LowMemoryDetector.java

```diff
@@ -185,11 +163,11 @@ public void logGCInfo(AccumuloConfiguration conf) {
         return;
       }
 
-      if (maxIncreaseInCollectionTime > keepAliveTimeout) {
-        Halt.halt("Garbage collection may be interfering with lock keep-alive.  Halting.", -1);
-      }
+      // if (maxIncreaseInCollectionTime > keepAliveTimeout) {
```

Comment:
I think this should be uncommented.

Reply:
Fixed in e58cfec34c - required other code to be restored too.